examples/example_experiment/example_experiment.cu

```diff
@@ -201,9 +201,9 @@ int main(int argc, char **argv){
 
   // add kernels that will carry out the sparse matrix vector multiplication
   // init random vals large enough for all the files
-  auto vals = new float[1049866];
-  fill_r(vals, 1049866);
-  auto vals_v = Array<float>(1049866, vals);
+  auto vals = new float[2*1049866];
```

Comment:
Shouldn't this be the number of rows in dblp? i.e. 317080?

Reply:
nope its the number of non zeros.